src/cmds/scripts/pbs_topologyinfo.py

```diff
@@ -48,20 +48,21 @@ def reportsockets_win(f):
     temp = f.read().split(',')
     for item in temp:
         if item.find('sockets:') != -1:
-            sockets = int(item.replace('sockets:', ''))
+            sockets = int(item[len('sockets:'):])
         if item.find('gpus:') != -1:
-            gpus = int(item.replace('gpus:', ''))
-    return (sockets, gpus, 0)
+            gpus = int(item[len('gpus:'):])
+        if item.find('mics:') != -1:
+            coproc = int(item[len('mics:'):])
+    return (sockets, gpus, coproc)
 
 
 def latest_hwloc(hwlocVersion):
-    if hwlocVersion.count('.') > 1:
-        hwlocVersion = hwlocVersion[:hwlocVersion.rfind('.')]
-    hwlocVersion = float(hwlocVersion)
-    if hwlocVersion >= 1.11:
+    hwlocVersion = hwlocVersion.split('.')
+    major = int(hwlocVersion[0])
+    minor = int(hwlocVersion[1])
```

Comment:
Can you change this to: minor = int(hwlocVersion[1]) if len(hwlocVersion) > 1 else 0